### ISSUE TITLE
Consider removing 'cross-var' as a dependency

### ISSUE THREAD

**Issue by bass3l** [Mermade/reslate#57]
**Bug Description**
'cross-var' has been unmaintained for more than 3 years now, I just came across a weird bug that prevents `npx reslate ...` from running correctly when there is a parent directory named `src` in the whole hierarchy. 

I'm posting the error description just in case some one came across it:

```
ReferenceError: Unknown plugin "transform-runtime" specified in.....
```

You can check the issue at `cross-var` here: https://github.com/elijahmanor/cross-var/issues/17

**Comment by MikeRalphson** [comment]
Thanks for reporting the bug and I agree, this issue will be helpful for anyone else hitting the same problem.

It's a pity the fork containing the fix isn't the slimmed-down non-babel version of `cross-var`.

Unless there is a better, maintained replacement for `cross-var` I'd be very reluctant to remove it, as it solves a real problem for us on the Windows platform.

**Comment by MikeRalphson** [comment]
Actually, because the bug is in the babel loader part of `cross-var`, switching to `cross-var-no-babel` should fix the issue.

**Comment by MikeRalphson** [comment]
Should be fixed in `v3.0.0-5`, but let me know if you still see a problem and I'll re-open.
